cmd/nomos/status/client.go

```diff
@@ -354,6 +358,18 @@ func (c *ClusterClient) namespaceRepoClusterStatus(ctx context.Context, cs *Clus
 	}
 }
 
+// IsOssInstallation will check for the existence of ConfigManagement object, Operator deployment, and root-reconciler deployment
+// If root-reconciler exist but ConfigManagement and Operator doesn't, it indicates an OSS installation
+func (c *ClusterClient) IsOssInstallation(ctx context.Context) bool {
+	v, _ := c.ConfigManagement.Version(ctx)
```

Comment:
The error returned by `c.ConfigManagement.Version` is swallowed. You need to better handle the error here.

Reply:
added error handling to ConfigManagement and made sure that the message that we need for OSS is `NOT INSTALLED`